docs/user-guide/etendo-copilot/bundles/dev-assistant.md

```diff
@@ -0,0 +1,95 @@
+
+2. The assistant will ask to the user for information to add, like columns, data types, help and description, etc.
+
+3. The user must confirm the steps or set up a modification about the given information.
+
+4. Recompile and restart Tomcat to a successful application of changes.
+
+5. The window can be viewed with the user role.
+
+
+
+
+## Usage Example 
+
+![dev-assistant.png](../../../assets/user-guide/etendo-copilot/bundles/dev-assistant/dev-assistant.png)
+
+**User**: "I want to create a window "Workers dates" that contains information about employees' work dates, use the prefix 'copdev'."
+
+**Copilot**: "Let's start by registering the table for "Workers Dates".
+Please provide the following for better definition:
+Table name in singular (e.g., WorkerDate, EmploymentDate, etc.).
+A valid prefix, which is 'copdev' in your case.
+Any specific data columns (except default columns) you want in the table initially.
+Optionally, if you have a specific class name in mind for this table, please provide it.
+Based on the provided prefix 'copdev' and the idea of the table, I'm suggesting the table name as WorkerDate. Let's proceed if the suggested name is acceptable to you?"
+
+**User**: "Proceed and add some useful columns."
+
+**Copilot**: "The table copdev_WorkerDate has been successfully created and registered in the system. Specific columns for Start Date, End Date, and Role have also been added to the table. Additionally, a window titled "Workers dates" has been created to manage this table's data. The necessary fields in the window's tab have been registered and synchronized as well.
+Summary of Steps Completed:
+Registered the copdev_WorkerDate table in the Application Dictionary.
+Created the copdev_WorkerDate table in the database.
+Added specific columns:
+Start Date (type: Date, not null, default: now())
+End Date (type: Date, can be null)
+Role (type: String, can be null)
+Registered the columns in the Application Dictionary.
+Synchronized the terminology.
+Created a window and tab to manage WorkerDate records.
+Registered fields in the window's tab.
+Synchronized the terminology again.
+Next Steps:
+Compile the project.
+Restart Etendo.
+This will ensure that all changes take effect, and you can start using the new table and window.
+Is there anything else you'd like to do or adjust?"
+
+
+## Results
+
+**With the task complete, it is necessary recompile with a smartbuild and restart Tomcat**
+
+![dev-assistant1.png](../../../assets/user-guide/etendo-copilot/bundles/dev-assistant/dev-assistant1.png)
+
```

Comment:
<img width="769" alt="image" src="https://github.com/etendosoftware/docs/assets/106354215/6b6b57b9-2885-4a5e-91f1-ec1d7d45d22d">

It looks bad, it must be because of the tab